src/Migrations/Version20201123210350.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20201123210350 extends AbstractMigration
```

Comment:
Rather than making a new migration, should we just change to earlier migration that creates these?

Reply:
I thought about it too, but then we need to change the data migration sql scripts. I feel this is more cleaner